src/main/resources/biome-maps/thorium_v1.18.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "value": 2,
```

Comment:
we had this random between 1  and 2 now its a set value. Not sure if this can be done with the current implementation at all but i would like to keep it how it was